src/lib.rs

```diff
@@ -729,6 +729,30 @@ impl TestKit {
         self.create_block_with_tx_hashes(&tx_hashes);
     }
 
+    /// Creates block with the given transaction.
+    /// Transactions that are in mempool will be ignored.
+    ///
+    /// # Panics
+    ///
+    /// - Panics if given transaction has been already committed to the blockchain.
+    pub fn create_block_with_transaction<T: Transaction>(&mut self, tx: T) {
+        if !tx.verify() {
+            return;
```

Comment:
C++ heritage ? 😏  I try to avoid a `return` operator in Rust

Reply:
It's controversial, but I think that early return statements - when they help to avoid unneeded indentations - are generally considered idiomatic in Rust.